setup.py

```diff
@@ -25,11 +25,12 @@ def get_reqs(lookup=None, key="INSTALL_REQUIRES"):
         module_meta = module[1]
         if "exact_version" in module_meta:
             dependency = "%s==%s" % (module_name, module_meta["exact_version"])
-        elif "min_version" in module_meta:
-            if module_meta["min_version"] is None:
-                dependency = module_name
-            else:
+        else:
```

Comment:
It looks like the nesting of if else might be off here?

Reply:
This should be what you want, an elif

```python
# Read in requirements
def get_reqs(lookup=None, key="INSTALL_REQUIRES"):
    """get requirements, mean reading in requirements and versions from
    the lookup obtained with get_lookup"""

    if lookup is None:
        lookup = get_lookup()

    install_requires = []
    for module in lookup[key]:
        module_name = module[0]
        module_meta = module[1]
        if "exact_version" in module_meta:
            dependency = "%s==%s" % (module_name, module_meta["exact_version"])
        elif "min_version" in module_meta:
            if module_meta["min_version"] is None:
                dependency = module_name
            else:
                dependency = "%s>=%s" % (module_name, module_meta["min_version"])
        elif "max_version" in module_meta:
            if module_meta["max_version"] is None:
                dependency = module_name
            else:
                dependency = "%s<=%s" % (module_name, module_meta["max_version"])
        install_requires.append(dependency)
    return install_requires
```

Reply:
Interesting, it works for setting up SHPC. I had a similar code to yours before I then went reducing the lines down to just an if else if else. I am not familiar with elif logic - but it works!